### PR TITLE
[Hackathon] Added notebook and utils for AI photo painting

### PR DESCRIPTION
**Pull request by ntk97**
Added an AI photo painting notebook utilizing CocosNet and OpenVINO to synthesize a photo-realistic image based on exemplar image sketch semantic.

Signed-off-by: sarthakdeva-intel [sarthak.deva@intel.com](mailto:sarthak.deva@intel.com)
Signed-off-by: shauryad15-intel  [shaurya.dwivedi@intel.com](mailto:shaurya.dwivedi@intel.com)
Signed-off-by: Vishwas R [vishwas.r@intel.com](mailto:vishwas.r@intel.com)
Signed-off-by: ntk [nayana.tk@intel.com](mailto:nayana.tk@intel.com)
Signed-off-by: aabhish1 [abhishek.a@intel.com](mailto:abhishek.a@intel.com)

### CODE REVIEW

File: notebooks/408-cocosnet-model/408-cocosnet-ai-painting.ipynb [hunk 1]
```diff
@@ -0,0 +1,364 @@
+{
```
Comment (eaidova) [inline notebooks/408-cocosnet-model/408-cocosnet-ai-painting.ipynb:1]: <p>please provide more description e.g.</p><p> which model used,</p><p>diagram with pipeline work and its description</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (eaidova) [inline notebooks/408-cocosnet-model/408-cocosnet-ai-painting.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.    import gradio as gr
</pre><p>user can have not installed gradio as it is not listed in our requirements</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (eaidova) [inline notebooks/408-cocosnet-model/408-cocosnet-ai-painting.ipynb:1]: <p>please provide link for downloader documentation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (eaidova) [inline notebooks/408-cocosnet-model/408-cocosnet-ai-painting.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #23.                # TODO: Remove this snippet if reference and input semantics are not required
</pre><p>are you plan to remove TODO?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (eaidova) [inline notebooks/408-cocosnet-model/408-cocosnet-ai-painting.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #18.    demo.launch(share=True)
</pre><p>Is it possibly to use local instance if it Is applicable like in https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/237-segment-anything</p><p>At least I think it is better to remove this cell output (now any person can run demo on your machine using provided link)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Comment by eaidova** [comment]
@ntk97 just freandly remind that your contribution should follow [contribution guide](https://github.com/openvinotoolkit/openvino_notebooks/blob/main/CONTRIBUTING.md)

at the current moment, I see that this requirement does not met https://github.com/openvinotoolkit/openvino_notebooks/blob/main/CONTRIBUTING.md#readmes

**Comment by AnishaUdayakumar** [comment]
Thank you for your contribution. Please do the self-check first (copy the list and tick the item when it's done):

- [ ] each function is described by docstrings and type hints
- [ ] notebook contains explicit descriptions and explanatory diagrams
- [ ] the notebook doesn't use any data (image, video, etc.) that is not CC4.0 licensed 
- [ ] there is a README.md file in consistent style (look at other notebooks)
- [ ] the notebook is added to the main README
- [ ] there are no grammar, punctuation or typo issues (use any free tool for that)
- [ ] there are no committed files besides notebook and readme (please use images or videos from data dir)
- [ ] your PR doesn't change any other notebooks
- [ ] all CI checks passed

File: requirements.txt [hunk 2]
```diff
@@ -29,6 +29,9 @@ ipywidgets
 ipykernel>=5.*
 ipython>=7.16.3
 
+# AI Painting notebook requirements
+gradio==3.28.0
```
Comment (eaidova) [inline requirements.txt:33]: please install gradio inside notebook

**Pull request closed without merging**